packages/postgres-database/src/simcore_postgres_database/migration/versions/e3334cced752_new_products_prices_table.py

```diff
@@ -0,0 +1,48 @@
+"""new products_prices table
+
+Revision ID: e3334cced752
+Revises: 624a029738b8
+Create Date: 2023-09-15 00:25:18.116227+00:00
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "e3334cced752"
+down_revision = "624a029738b8"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "products_prices",
+        sa.Column("product_name", sa.String(), nullable=False),
+        sa.Column("usd_per_credit", sa.Numeric(scale=2), nullable=False),
```

Comment:
Question: I guess this also means that 0.34 $ is accepted. If they want to have prices to be more raffled to something say like 0.333$ that is not supported right?

Reply:
both are supported. the scale is used to round off in operations